website/docs/cdktf/cli-reference/commands.mdx

```diff
@@ -130,27 +130,30 @@ $ cdktf deploy --help
+      --log-level                               Which log level should be written. Only supported via setting the env CDKTF_LOG_LEVEL                                                                                                                                                      [string]
+  -a, --app                                     Command to use in order to execute cdktf app                                                                                                                                                                                             [required]
+  -o, --output                                  Output directory for the synthesized Terraform config                                                                                                                                                             [required] [default: "cdktf.out"]
+      --auto-approve                            Auto approve                                                                                                                                                                                                             [boolean] [default: false]
+      --outputs-file                             Path to file where stack outputs will be written as JSON                                                                                                                                                                                   [string]
+      --outputs-file-include-sensitive-outputs   Whether to include sensitive outputs in the output file                                                                                                                                                                  [boolean] [default: false]
+      --ignore-missing-stack-dependencies       Don't check if all stacks specified in the command have their dependencies included as well                                                                                                                              [boolean] [default: false]
+      --parallelism                             Number of concurrent CDKTF stacks to run. Defaults to infinity, denoted by -1                                                                                                                                                [number] [default: -1]
+      --refresh-only                            Select the "refresh only" planning mode, which checks whether remote objects still match the outcome of the most recent Terraform apply but does not propose any actions to undo any changes made outside of Terraform.  [boolean] [default: false]
+      --terraform-parallelism                   Customize number of parallel graph traversals by Terraform. By default, the parallelism flag is not forwarded to Terraform. Note: This flag is not supported by remote / cloud backend                                       [number] [default: -1]
```

Comment:
@laurapacilio I've added a new flag with some documentation, followed by a warning to differentiate between `parallelism` and the new `terraform-parallelism` flag. Can I ask for you to take a look over this change please?

Reply:
yes! I will put it on my calendar to look at this week! I just need a few days because everything is tight schedule-wise before HashiConf :)